assets/js/stripe-payment-request.js

```diff
@@ -11,6 +11,14 @@ jQuery( function( $ ) {
 	 * Object to handle Stripe payment forms.
 	 */
 	var wc_stripe_payment_request = {
+		/**
+		 * Indicates if the user has picked or entered a shipping address on the payment dialog.
+		 *
+		 * @since 5.3.0
+		 * @type {boolean}
+		 */
+		hasShippingAddress: false,
```

Comment:
done a797e79

Reply:
@asumaran JSDoc comments were kept still.

```
/**
 * Indicates if the user has picked or entered a shipping address on the payment dialog.
 *
 * @since 5.3.0
 * @type {boolean}
 */
```